x-pack/plugins/alerting/server/rules_client/tests/disable.test.ts

```diff
@@ -210,12 +201,9 @@ describe('disable()', () => {
         schedule: { interval: '10s' },
         alertTypeId: 'myType',
         enabled: false,
-        meta: {
```

Comment:
hmmm .... why are we no longer returning this?  Maybe we only returned it when creating a new key?  Seems like we should always be returning it.

Reply:
i removed that before i put  `...(!attributes.apiKeyOwner && { apiKeyOwner: null }),` back.
updateMeta function is like:
```
if (alertAttributes.hasOwnProperty('apiKey') || alertAttributes.hasOwnProperty('apiKeyOwner')) {
      alertAttributes.meta = alertAttributes.meta ?? {};
      alertAttributes.meta.versionApiKeyLastmodified = this.kibanaVersion;
    }
    return alertAttributes;
```

since we don't send neither apiKey nor apiKeyOwner it doesn't return any meta object...
with `...(!attributes.apiKeyOwner && { apiKeyOwner: null }),` it may work though...
Not sure if should i add it back.

Reply:
added back :)